src/app/services/plan.js

```diff
@@ -83,6 +83,7 @@ module.exports = (Model, App) => {
   const getIndividualPlan = async (userEmail, userId) => {
     const subscriptionPlans = (await stripeService.getUserSubscriptionPlans(userEmail, userId))
       .filter((subscription) => subscription.status === 'active' || subscription.status === 'trialing')
+      .filter((subscription) => subscription.type === 'individual')
```

Comment:
I approved it but please @sg-gs @masterprog-cmd  confirm this line does not break something as we do not know if all the plans have this metadata.

Reply:
I don't think so. The current items we have in the metadata of a plan are: planType, maxSpaceBytes, show, recurrence, name.

We should add it to all individual plans before deploying if I'm not mistaken, right @sg-gs ?

Reply:
This field is custom so, it could break the listing plans functionality, yes. I prefer just to add the strictly required metadata things on Stripe, as when a new product is added, etc, we have issues with that. We should be also trying to avoid adding code to this project. 

I think adding it to payments would be the correct thing, even under a custom collection, so we can control these things in a centralized out-of-non-technical-user-hands way.